03/app.js

```diff
@@ -1,3 +1,27 @@
 export default function randomNumber(min, max) {
+  if (typeof min === "undefined" || typeof max !== "number") {
+    return 0;
+  }
 
-}
\ No newline at end of file
+  if (max < 0 || min < 0) {
+    return 0;
+  }
+
+  if (typeof min !== 'number' || typeof max !== 'number') {
+    throw new Error(
+        'Property have to be a number'
+    );
+}
+
+  if (min === 1 && max === 1) {
+    return 1;
+  }
```

Comment:
Raczej bym nie pisał takiego warunku (przynajmniej nie po refaktoryzacji).
Jak już to `if(min === max) return min`